### PR TITLE
Minor UI Updates

### PR DESCRIPTION
**Pull request by emichaud998**
Pull requests into this repository require the following checks to be completed by the submitter and reviewers.

Changes in this PR:
Made some minor changes and fixes to the Lantern shiny dashboard UI such as:

- On the values page, made the selected number of entries shown in the table persist when you filter the table
- Turned off the multiselect on tables since it doesn't do anything
- Removed the footer on the "About Lantern" page, but kept the email
- Changed sentence on "About Lantern" page to make it more clear
- Added note on Validation page that the "messagingEndptRule" is not broken, it's an issue with the invariant itself

**Submitter:**
- [x] Administrative tasks are complete.
  - This pull request describes why these changes were made.
  - The JIRA ticket links to this PR.
  - The JIRA ticket for this PR is: [Lantern 461 JIRA Ticket](https://oncprojectracking.healthit.gov/support/browse/LANTERN-461)
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.
- [x] Code review has been performed.
  - Code diff has been done and checked to ensure only expected code is being committed.
  - A linter for the language has been run.

**Primary Reviewer:**

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code is maintainable, reusable, efficient, and correct.
  - Code accomplishes the tasks purpose.
  - Code follows style guidance appropriate for the language, including passing linter checks.
  - Code is well commented (which does not mean verbosely commented).
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.

**Merger:**

Prior to merging, ensure that all checks have been completed. Delete the branch after the merge
is complete.

### CODE REVIEW

File: shinydashboard/lantern/modules/validationsmodule.R [hunk 1]
```diff
@@ -39,7 +39,10 @@ validationsmodule_UI <- function(id) {
         htmlOutput(ns("anchorpoint")),
         htmlOutput(ns("failure_table_subtitle")),
         reactable::reactableOutput(ns("validation_failure_table")),
-        p("A green check icon indicates that an endpoint has successfully returned a Conformance Resource/Capability Statement. A red X icon indicates the endpoint did not return a Conformance Resource/Capability Statement.")
+        p("A green check icon indicates that an endpoint has successfully returned a Conformance Resource/Capability Statement. A red X icon indicates the endpoint did not return a Conformance Resource/Capability Statement."),
+        p("*Note: The messagingEndptRule is not broken, there is an issue with the Capability Statement invarient ", a("(cpb-3).", href = "http://hl7.org/fhir/capabilitystatement.html#invs", target = "_blank"),
```
Comment (bmath10) [inline shinydashboard/lantern/modules/validationsmodule.R:43]: invariant instead of invarient

**Comment by bmath10** [comment]
I think we should move the messagingEndptRule note to right under "The ONC Final Rule requires endpoints to support FHIR..."

**Comment by bmath10** [comment]
I know this isn't part of this ticket, but could we add that feature of the Endpoint page where the list of more than 5 organization resources gets put into a modal for the SMART Response and Security pages too? Cause right now for the NextGen endpoints it looks completely ridiculous lol

**Comment by emichaud998** [comment]
> The values page seemed to be resetting the number of entries in the table when I selected various dropdown options?

You are right I missed this table, the security table, and the smart response table somehow! Made changes to include these tables

File: shinydashboard/lantern/modules/validationsmodule.R [hunk 2]
```diff
@@ -24,7 +24,10 @@ validationsmodule_UI <- function(id) {
     ),
     fluidRow(
       column(width = 12,
-        p("The ONC Final Rule requires endpoints to support FHIR version 4.0.1, but we have included all endpoints for reference")
+        p("The ONC Final Rule requires endpoints to support FHIR version 4.0.1, but we have included all endpoints for reference"),
+        p("*Note: The messagingEndptRule is not broken, there is an issue with the Capability Statement invariant ", a("(cpb-3).", href = "http://hl7.org/fhir/capabilitystatement.html#invs", target = "_blank"),
+        "The invarient states that the Messaging endpoint has to be present when the kind is 'instance', and Messaging endpoint cannot be present when kind is NOT 'instance', but the FHIRPath expression is \"messaging.endpoint.empty() or kind = 'instance'\", which
```
Comment (bmath10) [inline shinydashboard/lantern/modules/validationsmodule.R:29]: "invariant"

**Comment by bmath10** [comment]
While I'm in the process of asking for very minor changes, could you fix the page header for the Implementation Guide page? Its in `server.R`, it accidentally says "Implmentation Guide"

**Review by bmath10 — APPROVED** [review]

**Pull request merged**